src/alarms/notifications/NewNotification.js

```diff
@@ -81,20 +78,62 @@ async function fetchNestedAssets(assetType, assetId) {
     )
       .then(response => response.json())
       .then(data => {
-        console.log("NewNotification fetchNestedAssets data", data);
-        console.log(
-          "NewNotification fetchNestedAssets data.filters",
-          data.filters
-        );
-        return data["filters"];
+        let nestedAssetsList = [];
+        if (data.filters) {
+          nestedAssetsList = data.filters;
+        } else if (data.pumps) {
+          nestedAssetsList = data.pumps;
+        }
+        return nestedAssetsList;
```

Comment:
Deze zeven regels kun je ook schrijven als `return data.filters || data.pumps;`, of de hele functie als `data => (data.filters || data.pumps)`.